WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -0,0 +1,122 @@
+import SwiftUI
+import DesignSystem
+
+extension NotificationsTableViewCell {
+    struct Content: View {
+        private let title: String
+        private let description: String?
+        private let shouldShowIndicator: Bool
+        private let avatarStyle: AvatarsView.Style
+        private let actionIconName: String? // TODO: Will be refactored to contain the action.
+
+        init(
+            title: String,
+            description: String?,
+            shouldShowIndicator: Bool,
+            avatarStyle: AvatarsView.Style,
+            actionIconName: String?
+        ) {
+            self.title = title
+            self.description = description
+            self.shouldShowIndicator = shouldShowIndicator
+            self.avatarStyle = avatarStyle
+            self.actionIconName = actionIconName
+        }
+
+        var body: some View {
+            HStack(alignment: .top, spacing: 0) {
+                avatarHStack
+                textsVStack
+                    .offset(x: -avatarStyle.leadingOffset*2)
+                    .padding(.horizontal, Length.Padding.split)
+                if let actionIconName {
+                    actionIcon(withName: actionIconName)
+                }
+            }
+            .padding(.trailing, Length.Padding.double)
+        }
```

Comment:
Not sure where to post this comment, but the existing row handles a scenario where the underlying comment is either Trashed or Spammed. The notifications looks like this:

![CleanShot 2024-02-08 at 18 51 31](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/2e2993a7-e167-4865-8943-e8829d89380a)

You can force display this state by making the following code changes:

![CleanShot 2024-02-08 at 18 51 15](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/6ac5c3a0-3a20-43c4-82f7-ed9c4946996c)

I think the new row should handle this state as well. Let me know what you think.

Reply:
On it!

Reply:
Resolved #https://github.com/wordpress-mobile/WordPress-iOS/pull/22564/commits/5302cffd483854ac4a22504402b047be121755b9